static/web.js

```diff
@@ -193,24 +193,29 @@ function share(result, version, code, button) {
     link.className = "shortening-link";
     result.firstChild.appendChild(link);
 
+    function repaint() {
+        // Sadly the fun letter-spacing animation can leave artefacts in at
+        // least Firefox, so we want to manually trigger a repaint. It doesn’t
+        // matter whether it’s relative or static for now, so we’ll flip that.
+        result.parentNode.style.visibility = "hidden";
+        result.parentNode.offsetHeight;
+        result.parentNode.style.visibility = "";
+    }
+    var repainter = setInterval(repaint, 50);
     request.onreadystatechange = function() {
+        clearInterval(repainter);
```

Comment:
Should this only be cleared when the readyState is 4? (i.e. when the request is actually completed?)